src/Mono.Android/Android.Views/View.cs

```diff
@@ -29,6 +29,20 @@ public T FindViewById<T> (int id)
 			return this.FindViewById (id).JavaCast<T> ();
 		}
 
+#if NET7_0_OR_GREATER || (NET6_0_OR_GREATER && ANDROID_33) || ANDROID_34
+		// See: https://cs.android.com/android/platform/superproject/+/master:frameworks/base/core/java/android/view/View.java;l=25322
+		public T RequireViewById<T> (int id)
+			where T : Android.Views.View
+		{
+			var view = FindViewById<T> (id);
```

Comment:
Shouldn't `View.RequireViewById<T>()` call `View.RequireViewById()`?

```csharp
return RequireViewById (id)
    .JavaCast<T>();
```

Reply:
If we want to have control over what the error message is, we need to call `FindViewById<T>()` and null check.

Reply:
`RequireViewById` is also only API 28 and higher. So we're also making a C# version that works all the way back.